### ISSUE TITLE
Setup ACLs on pitestdatasa to restrict access to just PHDI dev team

### ISSUE THREAD

**Issue by NatHillardUSDS** [CDCgov/prime-reportstream#4717]
**Description**
The folders within the new gen2 storage account `pitestdatasa` need to be locked down according to [this guide](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-access-control-model) using per-folder ACL permissions. Specifically, the `bronze`, `silver`, and `gold` containers need to be restricted just to: 
1) members of our dev team
2) you all on the devops team, and 
3) the service principal. 

While we could do this in the UI, it [looks like](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_data_lake_gen2_filesystem) terraform supports this and this would be a better solution. 

**Specific Requested resources**
`pitestdatasa` within `prime-ingestion-test`

**Existing Resources?**
See above

**Additional Context**
The state of VA has requested that we lock down our PII to just members of our development team, currently others outside of this team theoretically have access (though according to access logs have not attempted to access)

**Comment by JosiahSiegel** [comment]
ACL vs access keys findings: https://semanticbits.slack.com/archives/C02TRFPBFJN/p1647018707448329?thread_ts=1646938334.495719&cid=C02TRFPBFJN

**Comment by cwinters-usds** [comment]
@JosiahSiegel I think the work for this is already done?

**Comment by JosiahSiegel** [comment]
Coding is complete and restrictions applied to dev resources group. Waiting on permissions to apply to test resource group.

**Comment by JosiahSiegel** [comment]
[PR](https://github.com/CDCgov/prime-public-health-data-infrastructure/pull/57) and request complete
